.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do two jobs; lint and test. Lint will do a clean install of node dependencies and lint the source code. Test will do a install of node dependencies, install a specific version of chai and run tests across different versions of node and chai
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+    push:
+        branches: master
+    pull_request:
+        branches: master
+
+jobs:
+    lint:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: latest
```

Comment:
can we use a matrix here? so we can use 18, 20 and latest like chai

Reply:
Yes, I started with a copy from chai's file but I wanted to minimize the changes as much as possible. So this currently follows what would have been done in Travis. 
I want to do additional PRs after this one for things like; updating dependencies to more recent versions, dropping support for unsupported Node versions and adding chai v5 to the peer dependencies and CI.

Reply:
take note of the small flood of PRs i just dropped FYI, so we don't duplicate work (though i'm done now i think)

& makes sense, feel free to leave this one as is and we can just bump it in a separate PR